flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CEPMigrationTest.java

```diff
@@ -240,4 +270,159 @@ public Byte getKey(Event value) throws Exception {
 			return 0;
 		}
 	}
-}
+
```

Comment:
Is this the complete code that can be used to regenerate the snapshot files?

I think `WindowOperatorMigrationTest.testRestoreSessionWindowsWithCountTriggerFromFlink11()` shows how the code that generates the snapshot and the testing code can be interleaved nicely. This format also makes it easy (hopefully, as I will soon find out because I'm re-writing the tests for Flink 1.3 (https://issues.apache.org/jira/browse/FLINK-5969)) to update/adapt the tests on version updates of Flink.